code/modules/projectiles/ammunition/ballistic/shotgun.dm

```diff
@@ -99,6 +99,22 @@
 	pellets = 5
 	variance = 35
 
+/obj/item/ammo_casing/shotgun/uraniumpenetrator
+	name = "depleted uranium slug"
+	desc = "A relatively low-tech shell that makes use of refined uraniums' unique ability to self sharpen upon impact, giving it \
+	very impressive armor penatration capabilites."
+	icon_state = "dushell" 
+	projectile_type = /obj/item/projectile/bullet/shotgun_uraniumslug
+
+/obj/item/ammo_casing/shotgun/cryoshot
+	name = "cryoshot shell"
+	desc = "A state of the art shell that uses the cooling power of Rhigoxane to chill a target to the bone, while leaving them \
+	relatively unharmed."
```

Comment:
```suggestion
	them much harm."
```

Reply:
The sentence doesn't doesn't make sense without the addendum change, unfortunately

"to snap freeze a target, without causing  \
	relatively unharmed"

Reply:
oh I SEE NOW. I didnt realize it was for the first comment.